Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:21-jdk
+
+WORKDIR /app
+
+COPY ./build/libs/*.jar ./rentcar.jar
```

Comment:
u have few `*.jar` files in the build dir. What would be copied?

Reply:
the first one